app/src/main/java/org/fossasia/openevent/app/contract/model/IEventRepository.java

```diff
@@ -1,18 +1,22 @@
 package org.fossasia.openevent.app.contract.model;
 
+import org.fossasia.openevent.app.data.models.Attendee;
 import org.fossasia.openevent.app.data.models.Event;
 import org.fossasia.openevent.app.data.models.User;
 
 import java.util.List;
 
 import io.reactivex.Observable;
 
-public interface IEventModel {
+public interface IEventRepository {
```

Comment:
Yeah that was what I was trying to tell. A repository may contain several data sources, disk cache and network, so that's why we can't name its interface to be DataSource